### DISCUSSION TITLE
Quickstart: "Is slower" is actually faster. Why?

### DISCUSSION THREAD

**Discussion by tornikeo** [jax-ml/jax#9625] [Q&A]
Can anyone explain how do these results from docs->quickstart make any sense?

![image](https://user-images.githubusercontent.com/56438982/154583316-dcdb70c8-37d6-4b99-acac-a3bebe81cf14.png)

How is the second cell slower? It actually shows otherwise **right** there. It's faster by 50ms or so. Should we update the quickstart page?

**Comment by jakevdp** [comment]
Thanks for pointing this out. I believe it has to do with the fact that in numpy `x.T` is a zero-cost view of the original buffer, while in JAX outside of JIT-compilation, `x.T` results in allocation of a new buffer to hold the transpose.

We should probably update this example to make that more clear.
